src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/CashoutAggregate.cs

```diff
@@ -35,7 +35,8 @@ public class CashoutAggregate
             string hotWalletAddress, 
             string toAddress, 
             decimal amount, 
-            string assetId)
+            string assetId,
+            CashoutState state = CashoutState.Started)
```

Comment:
state still here, should be removed from parameters of this ctor

Reply:
fixed